x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -1400,6 +1400,13 @@ export const UPDATE_RULE_BUTTON = i18n.translate(
   }
 );
 
+export const UPDATE_RULE_BUTTON_TOOLTIP_CONFLICTS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.upgradeRules.button.conflicts',
+  {
+    defaultMessage: 'Rule has conflicts. Please resolve them manually.',
```

Comment:
Out of curiosity, how would users manually resolve a rule's field conflicts? 

```suggestion
    defaultMessage: 'Rule has conflicts. Resolve them manually.',
```